### PR TITLE
Added most recent algorithm

### PR DESCRIPTION
**Pull request by Erik-McKelvey**

### CODE REVIEW

File: enrollment_predictions/models/course_model.py [hunk 1]
```diff
@@ -225,7 +248,7 @@ def main():
     from sklearn.metrics import mean_squared_error
```
Comment (sispuella) [inline enrollment_predictions/models/course_model.py:248]: Please move imports to the top for formatting

**Review by sispuella — COMMENTED** [review]
Please move fix the formatting issue

File: enrollment_predictions/models/course_model.py (hunk 1 above)
Comment (Erik-McKelvey) [reply enrollment_predictions/models/course_model.py:248]: Will do this in the next PR

**Review by sispuella — APPROVED** [review]
ty

**Pull request merged**